docker/container.sh

```diff
@@ -231,6 +231,24 @@ x11_cleanup() {
     fi
 }
 
+submit_job() {
+
+    echo "[INFO] Arguments passed to job script ${@}"
+    case $CLUSTER_JOB_SCHEDULER in
+        "SLURM")
+            ssh $CLUSTER_LOGIN "cd $CLUSTER_ISAACLAB_DIR && sbatch $CLUSTER_ISAACLAB_DIR/docker/cluster/submit_job_slurm.sh \"$CLUSTER_ISAACLAB_DIR\" \"isaac-lab-$container_profile\" ${@}"
+            ;;
+        "PBS")
+            ssh $CLUSTER_LOGIN "cd $CLUSTER_ISAACLAB_DIR && bash $CLUSTER_ISAACLAB_DIR/docker/cluster/submit_job_pbs.sh \"$CLUSTER_ISAACLAB_DIR\" \"isaac-lab-$container_profile\" ${@}"
+            ;;
+        *)
+            echo "[ERROR] Unsupported job scheduler specified: $CLUSTER_JOB_SCHEDULER"
```

Comment:
Always a good practice to give informant error for the user to figure out what they should/could have done.

```suggestion
            echo "[ERROR] Unsupported job scheduler specified: '$CLUSTER_JOB_SCHEDULER'. Supported options are: ['SLURM', 'PBS']."
```